codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -260,6 +272,22 @@ public final DefaultStream parent() {
         }
 
         @Override
+        public final boolean isViable() {
```

Comment:
Boy, "writable" sure would be a good name .... (Nate giggles while sipping his tea) :)